Libraries/Components/ScrollView/RecyclerViewBackedScrollView.android.js

```diff
@@ -113,6 +113,18 @@ var RecyclerViewBackedScrollView = React.createClass({
       );
     });
 
+    const refreshControl = this.props.refreshControl;
+    if (refreshControl) {
+      // Wrap the NativeAndroidRecyclerView with a AndroidSwipeRefreshLayout.
+      return React.cloneElement(
+        refreshControl,
+        {style: props.style},
```

Comment:
I think the clearer way to do this is not include style in the `props` declaration (line 95). This way we can be explicit about what style we want for each case.

Should look like
``` js
   const recyclerProps = { ... };

   ...

    const refreshControl = this.props.refreshControl;
     if (refreshControl) {
       // Wrap the NativeAndroidRecyclerView with a AndroidSwipeRefreshLayout.
       return React.cloneElement(
         refreshControl,
         {style: [styles.base, this.props.style]},
         <NativeAndroidRecyclerView {...recyclerProps} style={styles.base}>
           {wrappedChildren}
         </NativeAndroidRecyclerView>
       );
     }
```
`styles.base` being `{flex: 1}`

We'll also have to change the case where there is no refreshControl to `style={[styles.base, this.props.style]}` and `{...recyclerProps}`